lib/Staticman.js

```diff
@@ -154,24 +241,60 @@ Staticman.prototype._getExtensionForFormat = function (format) {
     case 'yaml':
     case 'yml':
       return 'yml'
+
+    case 'frontmatter':
+      return 'md'
+  }
+}
+
+Staticman.prototype._processSubscriptions = function (fields, options) {
+  // Initialise SubscriptionsManager
+  const subscriptions = new SubscriptionsManager(this.github, this.rsa)
+
+  // If there isn't a parent or if notifications are not enabled, there's
+  // nothing to do here
+  if (!options.parent || !this.siteConfig.get('notifications.enabled')) {
```

Comment:
The code for notifications is still WIP.

In any case, the idea there is that if you want to subscribe to something, you need to send the ID of the comment you're subscribing to. That will come through as `options[parent]`.